src/include/mallocMC/mallocMC_constraints.hpp

```diff
@@ -80,8 +79,7 @@ namespace mallocMC{
     typename CreationPolicies::Scatter<x,y>,
     typename DistributionPolicies::XMallocSIMD<z> 
   >{
-    BOOST_MPL_ASSERT_MSG(x::pagesize::value == z::pagesize::value,
-        Pagesize_must_be_the_same_when_combining_Scatter_and_XMallocSIMD, () );
+    static_assert(x::pagesize == z::pagesize, "Pagesize_must_be_the_same_when_combining_Scatter_and_XMallocSIMD");
```

Comment:
could you please remove the underscores. `_`

Reply:
makes perfect sense!